Modules/ChequeModule/Views/ChequeView.xaml

```diff
@@ -17,17 +18,160 @@
         </ResourceDictionary>
     </UserControl.Resources>
 
```

Comment:
May try to reuse and create UI control styling at `ControlStyle` in the Resource project then apply it to the View.
This intention is to have a clean structure for better reading.